vectorlm/utils/model_utils.py

```diff
@@ -22,14 +22,45 @@
     PreTrainedModel,
     PreTrainedTokenizer,
 )
+from peft import (
+    LoraConfig,
+    LoraModel,
+    get_peft_model,
+    PeftModel,
+    PeftConfig,
+    PeftModelForCausalLM,
+)
 
+def load_peft_model_and_tokenizer(
+    path: str,
+    use_mp: bool,
+    use_fa: bool,
+    max_seq_len: int,
+    peft_adapter_path: str,
+    adapter_name: str = "default",
+    is_trainable: bool = False,
+    config: Optional[PeftConfig] = None,
+    **kwargs: Any
+) -> Tuple[PeftModel, PreTrainedTokenizer]:
+    """Loads a PEFT adapter to the base model and returns the PeftModel
+    E.g., a base llama-2-13b-chat-hf w/ adapter named nifty
+    ├── adapters_lora
+        ├── llama-2-13b-chat-hf+nifty
+    Args:
```

Comment:
Could you include all the arguments here? For the repeated ones, you can likely just copy paste them from `load_model_and_tokenizer`.